src/api/entities/Instruction/types.ts

```diff
@@ -48,6 +52,10 @@ export type InstructionDetails = {
    * Date at which the trade was executed (optional, for offchain trades)
    */
   valueDate: Date | null;
+  /**
+   * Venue to which the Instruction belongs to
+   * @note From 7.x chain, an Instruction can be created without specifying a Venue. Hence the null value.
```

Comment:
```suggestion
   * @note From 7.x chain, Instructions can be created without a Venue. Hence the possible null value.
```
(nit) To me what comes after "hence" should always be true, possible makes it more in line with "can" from the previous sentence.